be/src/service/starrocks_main.cpp

```diff
@@ -24,7 +24,6 @@
 #include <unistd.h>
 
 #include <boost/thread/thread.hpp>
```

Comment:
this can be replaced?

Reply:
Cannot be replaced directly, because there is no direct equivalent for thread_group is available in STL, maybe can be replace with std::vector\<std::thread\>, if so, i will handle this in another PR.